package.json

```diff
@@ -27,7 +27,6 @@
 		"build": "turbo run build",
 		"lint": "turbo run lint --parallel --continue",
 		"lint:fix": "turbo run lint --parallel --continue",
-		"prepare": "node -e \"try { require('husky').install() } catch (e) {if (e.code !== 'MODULE_NOT_FOUND') throw e}\"",
```

Comment:
This will be called by npm, its a lifecycle script: https://docs.npmjs.com/cli/v9/using-npm/scripts#life-cycle-scripts

This is what installs husky for pre-commit hooks.